verify.go

```diff
@@ -274,6 +274,17 @@ func (v *IDTokenVerifier) Verify(ctx context.Context, rawIDToken string) (*IDTok
 		if t.Expiry.Before(now()) {
 			return nil, fmt.Errorf("oidc: token is expired (Token Expiry: %v)", t.Expiry)
 		}
+
+		// If nbf claim is provided in token, ensure that it is indeed in the past.
+		if token.NotBefore != nil {
+			nbfTime := time.Time(*token.NotBefore)
+			nowTime := now()
+			leeway := 1 * time.Minute
+
+			if nowTime.Add(leeway).Before(nbfTime) {
+				return nil, fmt.Errorf("oidc: current time %d before the nbf (not before) time: %d", nowTime.Unix(), nbfTime.Unix())
```

Comment:
nit: for consistency with the token expiry above, just used %v for the time formatting

```
return nil, fmt.Errorf("oidc: current time %v before nbf (not before) time: %v", nowTime, nbfTime)
```

Reply:
Yeah I think there's a point to be made for printing the values in the same form as they come in the token, but consistency is key of course. Fixed.